torch/jit/frontend.py

```diff
@@ -571,7 +571,9 @@ def build_ExtSlice(ctx, base, extslice):
         base = build_expr(ctx, expr.value)
         sub_type = type(expr.slice)
         if sub_type is ast.Index:
-            if isinstance(expr.slice.value, ast.Tuple) or isinstance(expr.slice.value, ast.List):
+            if isinstance(expr.slice.value, ast.Tuple):
```

Comment:
If you are making a change to the python frontend, you probably need to make one to the string frontend as well. `self.checkScript` will make sure the behavior is the same between Python, TS+python, TS+no python.

Reply:
Btw I switched to use `self.checkScript` in the test and it passed. Is string frontend the one that takes in function strings and parse it? If so this change might be shared by both I think?